unit_tests/test_aggregator.py

```diff
@@ -99,137 +79,125 @@ def fixture_aggregator_from_env(mocker):
 
-    # *
     @staticmethod
-    def test_constructor__device_uuid_list(aggregator_explicit):
+    def test_constructor_device_uuid_list(aggregator_explicit):
         assert aggregator_explicit.device_uuid_list == []
 
-    # *
     @staticmethod
-    def test_constructor__aggregator_uuid(aggregator_explicit):
+    def test_constructor_aggregator_uuid(aggregator_explicit):
         assert aggregator_explicit.aggregator_uuid is None
 
     @staticmethod
-    def test_constructor__client_command_buffer():
+    def test_constructor_client_command_buffer():
         with patch("gsy_e_sdk.aggregator.ClientCommandBuffer") as mocked_class:
             Aggregator(aggregator_name=test_agg["name"])
             mocked_class.assert_called_once()
 
     @staticmethod
-    def test_constructor__connect_to_simulation():
+    def test_constructor_connect_to_simulation():
         with patch.object(Aggregator, "_connect_to_simulation") as mocked_method:
             Aggregator(aggregator_name=test_agg["name"])
             mocked_method.assert_called_once()
 
-    # *
     @staticmethod
-    def test_constructor__latest_grid_tree(aggregator_explicit):
+    def test_constructor_latest_grid_tree(aggregator_explicit):
         assert aggregator_explicit.latest_grid_tree == {}
 
-    # *
     @staticmethod
-    def test_constructor__latest_grid_tree_flat(aggregator_explicit):
+    def test_constructor_latest_grid_tree_flat(aggregator_explicit):
         assert aggregator_explicit.latest_grid_tree_flat == {}
 
-    # *
     @staticmethod
-    def test_constructor__area_name_uuid_mapping(aggregator_explicit):
+    def test_constructor_area_name_uuid_mapping(aggregator_explicit):
         assert aggregator_explicit.area_name_uuid_mapping == {}
 
 
 class TestStartWebsocketConnection:
     """Includes methods which test start_websocket_connection method."""
 
     @staticmethod
-    def test_start_websocket_connection__dispatcher(aggregator_explicit):
-        with patch("gsy_e_sdk.aggregator.AggregatorWebsocketMessageReceiver") \
-                as mocked_class:
-            aggregator_explicit.start_websocket_connection()
-            mocked_class.assert_called()
+    def test_start_websocket_connection_dispatcher(aggregator_explicit):
+        aggregator_explicit.start_websocket_connection()
+        assert aggregator_explicit.dispatcher is not None
 
     @staticmethod
-    def test_start_websocket_connection__websocket_creation(aggregator_explicit):
-        websocket_uri_test = f"{test_simulation['websockets_domain_name']}" \
-                             + f"/{test_simulation['uuid']}/aggregator/" \
-                             + f"{test_agg['uuid']}/"
+    def test_start_websocket_connection_websocket_creation(aggregator_explicit):
+        websocket_uri_test = (f"{test_simulation['websockets_domain_name']}"
+                              + f"/{test_simulation['uuid']}/aggregator/"
+                              + f"{test_agg['uuid']}/")
```

Comment:
We tend to use parenthesis instead of backslashes for line escaping, except for the assertions